http-common/src/test/java/com/wultra/core/http/common/headers/UserAgentTest.java

```diff
@@ -27,19 +31,90 @@
  */
 class UserAgentTest {
 
+    private static final String[] USER_AGENTS = new String[] {
+            "PowerAuthNetworking/1.1.7 (en; cellular) com.wultra.app.Mobile-Token.wultra_test/2.0.0 (Apple; iOS/16.6.1; iphone12,3)",
+            "PowerAuthNetworking/1.2.1 (uk; wifi) com.wultra.android.mtoken.gdnexttest/1.0.0-gdnexttest (samsung; Android/13; SM-A047F)",
+            "PowerAuthNetworking/1.1.7 (en; unknown) com.wultra.app.MobileToken.wtest/2.0.0 (Apple; iOS/16.6.1; iphone10,6)",
+            "PowerAuthNetworking/1.1.7 (en; wifi) com.wultra.app.MobileToken.wtest/2.0.0 (Apple; iOS/16.7.1; iphone10,6)"
```

Comment:
You can add default values from PowerAuth mobile SDK:

- `PowerAuth2TestsHostApp-ios/1.0 PowerAuth2/1.7.8 (iOS 17.0, simulator)` 
   - MainBundle/Version PowerAuth2/Version (iOS Version, deviceString)
- `"PowerAuth2/1.7.8 (Android 13, Google Pixel 4)"`
   - PowerAuth2/Version (Android Version, Build.MANUFACTURER Build.MODEL)

Reply:
Just add: For PowerAuth in our MobileToken App it's `MobileToken/1.2.0 PowerAuth2/1.7.8 (iOS 15.7.9, iPhone9,3)`.

Code in the SDK here:
https://github.com/wultra/powerauth-mobile-sdk/blob/4490ce91a48c62e64ef459f0838310377a639ba1/proj-xcode/PowerAuth2/PowerAuthSystem.m#L104

https://github.com/wultra/powerauth-mobile-sdk/blob/4490ce91a48c62e64ef459f0838310377a639ba1/proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/system/PowerAuthSystem.java#L32

Reply:
I have incorporated all the provided examples, thanks.